hardware_interface/src/components/component_lists_management.hpp

```diff
@@ -41,51 +40,38 @@ namespace components
   * list is is empty; return_type::OK otherwise.
   */
 inline return_type get_internal_values(
-  std::vector<double> & values, const std::vector<std::string> & queried_interfaces,
-  const std::vector<std::string> & int_interfaces, const std::vector<double> & int_values)
+  std::vector<double> & values, const std::vector<InterfaceInfo> & queried_interfaces,
+  const std::vector<InterfaceInfo> & int_interfaces, const std::vector<double> & internal_values)
 {
-  if (queried_interfaces.size() == 0) {
-    return return_type::INTERFACE_NOT_PROVIDED;
-  }
-
-  for (const auto & interface : queried_interfaces) {
-    auto it = std::find(
-      int_interfaces.begin(), int_interfaces.end(), interface);
-    if (it != int_interfaces.end()) {
-      values.push_back(int_values[std::distance(int_interfaces.begin(), it)]);
-    } else {
-      values.clear();
-      return return_type::INTERFACE_NOT_FOUND;
-    }
-  }
+  // TODO(andyz)
   return return_type::OK;
 }
 
 /**
  * \brief Set all internal values to to other vector. Return value is used for API consistency.
  *
  * \param values output list of values.
- * \param int_values internal values of the component.
+ * \param internal_values internal values of the component.
```

Comment:
Renamed from int_values to internal_values, because I think `integer` when I read `int`